src/rest-server/src/models/job.js

```diff
@@ -395,7 +453,15 @@ class Job {
         appCompletedTime: frameworkStatus.applicationCompletedTimestamp,
         appExitCode: frameworkStatus.applicationExitCode,
         appExitDiagnostics: frameworkStatus.applicationExitDiagnostics,
+        appExitMessages: {
+          contaierStderr: this.extractContainerStderr(frameworkStatus.applicationExitDiagnostics),
+          runtimeError: this.extractRuntimeOutput(frameworkStatus.applicationExitDiagnostics),
+        },
         appExitType: frameworkStatus.applicationExitType,
+        appExitTriggerMessage: frameworkStatus.applicationExitTriggerMessage,
```

Comment:
Can we remove this? Because it is already in appStaticExitInfo, and appStaticExitInfo is the ground truth